### PR TITLE
ACT-1163: Add PolicyTrackingGroup Resource

### PR DESCRIPTION
**Pull request by jtirjan**

### CODE REVIEW

File: src/sdk/resources/policyTrackingGroups.ts [hunk 1]
```diff
@@ -0,0 +1,15 @@
+import { IHttpClient } from '../http';
+import { PagedResource}  from './resource';
+import { PolicyTrackingGroup } from '../models';
+
+export class PolicyTrackingGroupsResource extends PagedResource<PolicyTrackingGroup> {
+	constructor(httpClient: IHttpClient, policyId?: string) {
```
Comment (smbecker) [inline src/sdk/resources/policyTrackingGroups.ts:6]: This should be a similar design to [here](https://github.com/portalarchitects/skysync-cli/blob/e022338957ee539ca837cbacac69148d65cedff4/src/sdk/resources/accountMaps.ts#L12) where `policyId` is required and this resource hangs off of the `PolicyResource`

**Review by smbecker — CHANGES_REQUESTED** [review]

File: src/sdk/resources/policyTrackingGroups.ts (hunk 1 above)
Comment (jtirjan) [reply src/sdk/resources/policyTrackingGroups.ts:6]: Ah! I knew there had to be another place where we were doing this. Thanks!

Comment (smbecker) [reply src/sdk/resources/policyTrackingGroups.ts:6]: Also `policies/tracking_groups` is not a valid route, even after https://github.com/portalarchitects/skysync-vnext/pull/3261 merges

Comment (jtirjan) [reply src/sdk/resources/policyTrackingGroups.ts:6]: It was recently. @mmitoff just made a change to use `/policies/{id}/tracking_groups` for all methods

**Review by smbecker — APPROVED** [review]

**Pull request merged**